docs/app-modules/interchain-accounts/auth-modules.md

```diff
@@ -210,7 +198,8 @@ packetData := icatypes.InterchainAccountPacketData{
     Data: data,
 }
 
-_, err = keeper.icaControllerKeeper.TrySendTx(ctx, chanCap, p, packetData)
+// Send the interchain accounts packet, returning the packet sequence
+seq, err = keeper.icaControllerKeeper.TrySendTx(ctx, chanCap, portID, packetData)
```

Comment:
I figured it was better to include the sequence return value in the code snippets for developer docs